tensorflow/tools/ci_build/mac/gpu/pip/run.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ==============================================================================
+#
+# Usage:
```

Comment:
The whl file built by build_pip_package is too specific, e.g.,:

Copied wheel file: tensorflow-0.11.0rc2-cp27-cp27m-macosx_10_11_x86_64.whl --> tensorflow-0.11.0rc2-py2-none-any.whl

If I remember correctly, if we don't do thisrenamin, our whl files will not install properly on all mac versions.